### PR TITLE
[Enhancement]: Improve the rendering of Docs-API

### PR DESCRIPTION
**Pull request by ruoningYu**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repositories?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

**Before PR**:

- [ ] I have read and followed the workflow indicated in the [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) to create this PR.
- [ ] Pre-commit or linting tools indicated in [CONTRIBUTING.md](https://github.com/open-mmlab/mmediting/blob/master/.github/CONTRIBUTING.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] If the modification has potential influence on downstream or other related projects, this PR should be tested with some of those projects.
- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: docs/en/dataset_zoo/0_overview.md [hunk 1]
```diff
@@ -1,8 +1,19 @@
 # Overview
 
 \\n- [Prepare Super-Resolution Datasets](./1_super_resolution_datasets.md)
```
Comment (Z-Fran) [inline docs/en/dataset_zoo/0_overview.md:3]: Please check dataset scripts, making this file looks correct

File: docs/en/api/generated/mmedit.apis.delete_cfg.rst [hunk 2]
```diff
@@ -0,0 +1,6 @@
+﻿mmedit.apis.delete\_cfg
```
Comment (Z-Fran) [inline docs/en/api/generated/mmedit.apis.delete_cfg.rst:1]: Don't upload/commit `api/generated` directory, like https://github.com/open-mmlab/mmengine/pull/469/files .

Comment (ruoningYu) [reply docs/en/api/generated/mmedit.apis.delete_cfg.rst:1]: Ok, I will remove these in the next commit

**Comment by ruoningYu** [comment]
Issues in dataset_zoo have been fixed and the api/generated has been removed

**Review by Z-Fran — DISMISSED** [review]

**Comment by Z-Fran** [comment]
1. resolve **lint error** of CI (docs/en/api/)
2. resolve conflicts @ruoningYu

**Review by Z-Fran — APPROVED** [review]

**Pull request merged**